be/src/runtime/stream_load/stream_load_context.cpp

```diff
@@ -70,6 +70,17 @@ std::string StreamLoadContext::to_json() const {
     writer.Int64(receive_bytes);
     writer.Key("LoadTimeMs");
     writer.Int64(load_cost_nanos / 1000000);
+    writer.Key("BeginTxnTimeMs");
+    writer.Int64(begin_txn_cost_nanos / 1000000);
+    writer.Key("StreamLoadPutTimeMs");
+    writer.Int64(stream_load_put_cost_nanos / 1000000);
+    writer.Key("ReadDataTimeMs");
+    writer.Int64(read_data_cost_nanos / 1000000);
+    writer.Key("WriteDataTimeMs");
+    writer.Int(write_data_cost_nanos / 1000000);
+    writer.Key("CommitAndPublishTimeMs");
+    writer.Int64(commit_and_publish_txn_cost_nanos / 1000000);
```

Comment:
modify the document of stream load to explain this new fields

Reply:
done